task_1.py

```diff
@@ -1,2 +1,14 @@
 def cross(n):
-    pass
+    w = 4
```

Comment:
Jak funkcja, która powinna działać dla każdego `n` zaczyna się od ustawienia na sztywno wartości 4 to raczej nie ma szans zadzialać (przejść testów).